File: fizzBuzz/fizzBuzz.go
package main

import (
	"fmt"
	"strconv"
)

func fizzBuzz(n int) []string {
	var answer []string
	for i := 1; i <= n; i++ {
		txt := ""
		if i%3 == 0 {
			txt += "Fizz"
		}
		if i%5 == 0 {
			txt += "Buzz"
		}
		if txt == "" {
			txt = strconv.Itoa(i)
		}
		answer = append(answer, txt)
	}
	return answer
}

func main() {
	fmt.Println(fizzBuzz(3))

}

File: fizzBuzz/fizzBuzz.py
def fizzBuzz(self, n: int) -> List[str]:
    answer = []
    i = 1
    j = ""
    while i <= n:
        if i % 3 == 0 and i % 5 == 0:
            j = "FizzBuzz"
        elif i % 5 == 0:
            j = "Buzz"
        elif i % 3 == 0:
            j = "Fizz"
        else:
            j = str(i)
        answer.append(j)
        i += 1
    return answer